test/functional/ex_cmds/checkhealth_spec.lua

```diff
@@ -0,0 +1,17 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear = helpers.clear
+local eq = helpers.eq
+local getcompletion = helpers.funcs.getcompletion
+
+describe(':checkhealth', function()
+  before_each(function()
+    clear()
+  end)
+
+  describe('completion', function()
```

Comment:
there is `test/functional/viml/completion_spec.lua` or `test/functional/plugin/health_spec.lua `

Reply:
Okay, I'll move it to `test/functional/plugin/health_spec.lua`.